documentation/en/cli-lotus.md

```diff
@@ -2820,6 +2821,49 @@ OPTIONS:
    --help, -h  show help
 ```
 
+### lotus f3 powertable
+```
+NAME:
+   lotus f3 powertable
+
+USAGE:
+   lotus f3 powertable command [command options] [arguments...]
+
+COMMANDS:
+   get, g              Get F3 power table at a specific instance ID or latest instance if none is specified.
+
+                       The instance may be specified as the first argument. If unspecified,
+                       the latest instance is used.
+   get-proportion, gp  Gets the total proportion of power for a list of actors at a given instance.
+
+                       The instance may be specified via --instance flag. If unspecified, the
+                       latest instance is used.
+
+                       The list of actors may be specified as Actor ID or miner address, space
+                       separated, via arguments. Example:
+                         $ lotus f3 powertable get-proportion -i 42 1413 t01234 f12345
+   help, h             Shows a list of commands or help for one command
+
+OPTIONS:
+   --help, -h  show help
+```
+
+#### lotus f3 powertable get
```

Comment:
here's a puzzle, why are we not getting a section with `## lotus f3 powertable get-proportion`?

Reply:
It's a bug: https://github.com/filecoin-project/lotus/issues/12705

Reply:
I have captured my thoughts on the right way to fix this here: https://github.com/filecoin-project/lotus/issues/12706

For now i work around it by avoiding new lines in usage.